packages/app/components/onboarding/hcaptcha/challenge.tsx

```diff
@@ -36,6 +36,12 @@ export const Challenge = () => {
   const captchaRef = useRef<ConfirmHcaptcha>(null);
 
   const showCaptcha = () => {
+    // we don't need captcha on e2e tests
+    if (process.env.E2E) {
```

Comment:
we should avoid changing code for tests. But I couldn't find any better way to do this. We can end the test on captcha screen though or create a dummy twitter/insta accounts.

Reply:
But LGTM anyway

Reply:
oh wow. There is? sure we should use that!